### PR TITLE
chore: Changes in user invited analytics event

### LINKED ISSUE #16510: [Bug]: Missing events in Audit logs
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

Following are the Events that are missing 

- [x] application.deployed
- [x] page.viewed
- [ ] user.invited (waiting for CE PR to merge)
- [x] query.executed (working fine locally)

### Steps To Reproduce

1. Go to home page 
2. click on members and setting tab 
3. under others tab click on Audit log

### Public Sample App

_No response_

### Version

http://ec2-65-2-149-212.ap-south-1.compute.amazonaws.com/applications

### PR DESCRIPTION
**Pull request by vishnu-gp**
## Description

This PR introduces slight changes in the way user invite analytics event is being sent.

Fixes #16510

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?

- Locally

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: app/server/appsmith-interfaces/src/main/java/com/appsmith/external/constants/AnalyticsEvents.java [hunk 1]
```diff
@@ -16,6 +16,7 @@ public enum AnalyticsEvents {
     LOGOUT,
     FIRST_LOGIN,
     EXECUTE_ACTION("execute_ACTION_TRIGGERED"),
+    EXECUTE_INVITE_USERS("execute_INVITE_USERS"),
```
Comment (trishaanand) [inline app/server/appsmith-interfaces/src/main/java/com/appsmith/external/constants/AnalyticsEvents.java:19]: Any reason for using the word `execute` here?

Comment (vishnu-gp) [reply app/server/appsmith-interfaces/src/main/java/com/appsmith/external/constants/AnalyticsEvents.java:19]: > Any reason for using the word `execute` here?

@trishaanand MixPanel data would get messed up if we change the name.

Comment (trishaanand) [reply app/server/appsmith-interfaces/src/main/java/com/appsmith/external/constants/AnalyticsEvents.java:19]: @vishnu-gp : What would be the event name visible in Audit Logs?

@Nikhil-Nandagopal : If we do end up changing names for longevity, how big of a change would it be for MixPanel?

Comment (vishnu-gp) [reply app/server/appsmith-interfaces/src/main/java/com/appsmith/external/constants/AnalyticsEvents.java:19]: @trishaanand In Audit Logs the name would be `user.invited`

Looping @vuiets as well for MixPanel

**Review by trishaanand — APPROVED** [review]

**Pull request merged**